src/source/PeerConnection/Rtcp.h

```diff
@@ -10,6 +10,28 @@ extern "C" {
 STATUS onRtcpPacket(PKvsPeerConnection, PBYTE, UINT32);
 STATUS onRtcpRembPacket(PRtcpPacket, PKvsPeerConnection);
 STATUS onRtcpPLIPacket(PRtcpPacket, PKvsPeerConnection);
+STATUS onRtcpTwccPacket(PRtcpPacket, PKvsPeerConnection);
+
+// https://tools.ietf.org/html/draft-holmer-rmcat-transport-wide-cc-extensions-01
+// Deltas are represented as multiples of 250us:
+#define TWCC_TICKS_PER_SECOND   (1000000LL / 250)
+#define MICROSECONDS_PER_SECOND 1000000LL
+
+#define IS_TWCC_RUNLEN(packetChunk)            ((((packetChunk) >> 15u) & 1u) == 0)
+#define TWCC_RUNLEN_STATUS_SYMBOL(packetChunk) (((packetChunk) >> 13u) & 3u)
+#define TWCC_RUNLEN_GET(packetChunk)           ((packetChunk) &0x1fffu)
+#define TWCC_IS_NOTRECEIVED(statusSymbol)      ((statusSymbol) == 0)
```

Comment:
Just a thought to make it cleaner. Can we use enums?  

```
typedef enum {
  TWCC_STATUS_SYMBOL_NOTRECEIVED = 0,
  TWCC_STATUS_SYMBOL_SMALLDELTA,
  TWCC_STATUS_SYMBOL_LARGEDELTA,
} TWCC_STATUS_SYMBOL
```

And then...

`#define TWCC_IS_RECEIVED(statusSymbol) ((statusSymbol) ==  (TWCC_STATUS_SYMBOL_SMALLDELTA) || (statusSymbol) == (TWCC_STATUS_SYMBOL_LARGEDELTA))`

Reply:
makes sense